makefile

```diff
@@ -0,0 +1,9 @@
+get-client-secret:
```

Comment:
I don't think that a makefile in the root of monorepo is a good idea. I mean, it relates to the example package, so it does not make sense to keep it in the root.

Reply:
similar to the scripts in package json, its a utility to avoid constant navigation to execute different commands. Removed for now!